README.md

```diff
@@ -211,6 +211,18 @@ You can chose to use an external command line (and thus save a line of space) by
 let fc['.*'] = { 'cmdline' : 'firenvim' }
 ```
 
+### Using a single neovim instance
+
+Firenvim can be made to use a single neovim instance. To do so, set the `server` setting to `'persistent'`. Firenvim will automatically start an instance on Firefox's startup and then launch a new one every time the previous one is `:quit`'ed. In this mode, every new Firenvim window is actually a Neovim floating window. This means that having the cursor move to another window/opening new floating windows can be pretty confusing and should be avoided.
```

Comment:
really cool!

NB: will hopefully be less confusing after tabgrid is finished. https://github.com/neovim/neovim/pull/7541